fcd/main.cpp

```diff
@@ -62,6 +64,9 @@ raw_ostream& llvm_errs()
 }
 #endif
 
+DECLARE_string(arch);
+DECLARE_string(os);
+
 namespace
 {
 	cl::opt<string> inputFile(cl::Positional, cl::desc("<input program>"), cl::Required, whitelist());
```

Comment:
Could you take this opportunity to convert all `llvm:cl`-based command-line flags into gflags ones?

Reply:
Sure, no problem. I could also run the file through clang-format or a similar tool, while I'm at it. Is that fine by you?